### PR TITLE
page level routing, homepage intro & imageList

### PR DESCRIPTION
**Pull request by pranta-barua007**
@coltonehrman 
applied image list from --> https://mui.com/components/image-list/
you can suggest other styles of image list from above

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+  {name: "Testimonials", route: "/testimonials"},
+  {name: "Contact", route: "/contact"}
 ];
 
 function App() {
   return (
     <>
       <NavBar pages={pages} />
-      <Hero />
+      <AppRoutes />
```
Comment (coltonehrman) [inline src/App.js:20]: I think we don't need a new component for this (yet)
It will be easier to read if the routes and `pages` are shown together

**Comment by coltonehrman** [comment]
I am seeing an inner slide bar around the image list
Screen Shot 2021-12-01 at 3 54 45 PM

**Comment by coltonehrman** [comment]
Looks like the prettier formatting didn't get applied and the max-width on the HeroImage didn't get applied, you might need to rebase your branch with the upstream

**Comment by pranta-barua007** [comment]
> Looks like the prettier formatting didn't get applied and the max-width on the HeroImage didn't get applied, you might need to rebase your branch with the upstream

shouldn't husky run before every  commit??
it doesn't ran on latest commit

**Comment by pranta-barua007** [comment]
> I am seeing an inner slide bar around the image list Screen Shot 2021-12-01 at 3 54 45 PM

because it has a fixed width for the container, if number of images get increased then it becomes scrollable

File: src/App.js (hunk 1 above)
Comment (pranta-barua007) [reply src/App.js:20]: > 

ok will move those into app.js

**Comment by pranta-barua007** [comment]
> > Looks like the prettier formatting didn't get applied and the max-width on the HeroImage didn't get applied, you might need to rebase your branch with the upstream
> 
> shouldn't husky run before every commit?? it doesn't ran on latest commit I did fetch the upstream yesterday which include husky + prettier config (told you about the issue yesterday on Discord!)
my branch is upto the date

Screenshot (75)

**Comment by pranta-barua007** [comment]
Screenshot 2021-12-02 115402

any guess why this happening?

**Pull request closed without merging**